docs/general/transaction-attacks.md

```diff
@@ -60,6 +60,22 @@ This will be showed in the Signer app and a careful user will notice it. If the
 Signer is already incorrect (or the Signer is corrupted) there is the risk of signing a non-intended
 transaction without the possibility of verifying it.
 
+### Replay Attack
+
+A replay attack is where past transactions can be replayed (same [balance](#balance-transfers),
+receiver account, etc.) without knowing private keys. This could happen in the context of
```

Comment:
I would say that that it's an attack where publicly known data can be "replayed" as a new extrinsic.